### PR TITLE
Scale test on 4 nvmeof GWs with 2048 namespaces

### PR DESCRIPTION
**Pull request by rahullepakshi**
Scale testing
Constants - 4 GWs, (1, 2, 4, 8, 16)subsystems in a test suite
variables - namespaces in scale steps of 400 to achieve 2048 incrementally

E2E 2 subsystems - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-U7EKDM/

### CODE REVIEW

File: suites/reef/nvmeof/tier-3_4-nvmeof-gw_8-sub_ns.yaml [hunk 1]
```diff
@@ -224,8 +224,159 @@ tests:
         run_io:
           - node: node8
             io_type: write
-      desc: Scale to 400 namespaces on 8 subsystems with IO
+      desc: Scale to 400 namespaces with IO
       destroy-cluster: false
       module: test_ceph_nvmeof_gateway_sub_scale.py
-      name: Scale to 400 namespaces on 8 subsystems with IO
+      name: Scale to 800 namespaces with IO on 4GW and 8 subsystems
```
Comment (HaruChebrolu) [inline suites/reef/nvmeof/tier-3_4-nvmeof-gw_8-sub_ns.yaml:230]: This should be 400 namespaces right?!

**Review by HaruChebrolu — APPROVED** [review]
lgtm

**Review by Manohar-Murthy — DISMISSED** [review]
merging based on team's review.

**Comment by rahullepakshi** [comment]
@Manohar-Murthy @HaruChebrolu changed tests as per discussion we had today on reducing scale steps. Please review again

**Review by HaruChebrolu — APPROVED** [review]
lgtm

**Review by Manohar-Murthy — APPROVED** [review]
lgtm.

**Pull request merged**